static/templates/user_info_popover_content.hbs

```diff
@@ -183,12 +183,14 @@
             </a>
         </li>
         {{/if}}
+        {{#if is_active}}
         {{#if show_manage_user_option}}
```

Comment:
I think we can add the check for active users in `show_manage_user_option` only.

Reply:
It's done.

Reply:
There is some confusion here. I meant that we can add the `is_active` condition when computing `show_manage_user_option` itself in `popovers.js`.

Reply:
oh, sorry for that, I got it now.

Reply:
Well, I did as you said, but `is_active` I find it useful for showing the option for `reactivate this user`, as I will just put `else` for condition `#if is_active`.
Or one more thing I can do here, do as you said in this commit, and do as I saying in the third commit(reactivate user), yes this third commit will undo these changes.

Reply:
Yes, I think that's fine, we would also need to rename `show_manage_user_option` as we will use that variable for rendering reactivate option too. In that case I think you can just keep this as it is and avoid changing it again the `reactivate user` commit.

Reply:
`reactivate user` is also a kind of managing user so what about renaming `show_manage_user_option` to just `can_manage_user`?

Reply:
That's fine I think.